src/argtable3.c

```diff
@@ -201,8 +203,15 @@ static char* alloc_shortoptions(struct arg_hdr** table) {
     result = xmalloc(len);
 
     res = result;
-    /* Add a leading + so getopt doesn't reorder argv */
-    *res++ = '+';
+    if (GETOPT_NOSHUFFLE == shuffle)
+    {
+        /* Add a leading + so getopt doesn't reorder argv */
+        *res++ = '+';
+    }
+    else
```

Comment:
If you do the above, then you have no need for this else because the compiler will complain that it's not one of the enums